types.go

```diff
@@ -2,20 +2,20 @@ package config
 
 // TemporalConfig defines Temporal configuration fields
 type TemporalConfig struct {
-	API          `json:"api,omitempty"`
-	Database     `json:"database,omitempty"`
-	IPFS         `json:"ipfs,omitempty"`
-	IPFSCluster  `json:"ipfs_cluster,omitempty"`
-	MINIO        `json:"minio,omitempty"`
-	RabbitMQ     `json:"rabbitmq,omitempty"`
-	AWS          `json:"aws,omitempty"`
-	Sendgrid     `json:"sendgrid,omitempty"`
-	Ethereum     `json:"ethereum,omitempty"`
-	Wallets      `json:"wallets,omitempty"`
-	APIKeys      `json:"api_keys,omitempty"`
-	Endpoints    `json:"endpoints,omitempty"`
-	Orchestrator `json:"orchestrator,omitempty"`
-	LogDir       string `json:"log_dir,omitempty"`
+	API         `json:"api,omitempty"`
+	Database    `json:"database,omitempty"`
+	IPFS        `json:"ipfs,omitempty"`
+	IPFSCluster `json:"ipfs_cluster,omitempty"`
+	MINIO       `json:"minio,omitempty"`
+	RabbitMQ    `json:"rabbitmq,omitempty"`
+	AWS         `json:"aws,omitempty"`
+	Sendgrid    `json:"sendgrid,omitempty"`
+	Ethereum    `json:"ethereum,omitempty"`
+	Wallets     `json:"wallets,omitempty"`
+	APIKeys     `json:"api_keys,omitempty"`
+	Endpoints   `json:"endpoints,omitempty"`
+	Nexus       `json:"Nexus,omitempty"`
```

Comment:
to be in line with the others:

```suggestion
	Nexus       `json:"nexus,omitempty"`
```

Reply:
good catch